### PR TITLE
Convert from truffle to hardhat

### PR DESCRIPTION
**Pull request by SalvoAgustin**
Created an empty hardhat project, runed yarn to create lock file, used contracts from zns branch, added .gitignore

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,87 +1,25 @@
...
-    "web-vitals": "^1.0.1",
-    "web3-react": "^5.0.5",
-    "web3modal": "^1.9.3"
+  "devDependencies": {
+    "@openzeppelin/contracts": "4.0.0",
+    "@openzeppelin/contracts-upgradeable": "4.0.0",
+    "@openzeppelin/hardhat-upgrades": "1.6.0",
+    "@openzeppelin/upgrades-core": "1.5.1",
+    "hardhat": "2.6.3",
+
```
Comment (Remscar) [inline package.json:8]: Why is there an empty line here?

**Review by Remscar — COMMENTED** [review]
I got this warning when installing dependencies:
```
warning " > @openzeppelin/hardhat-upgrades@1.6.0" has unmet peer dependency "@nomiclabs/hardhat-ethers@^2.0.0".
```

This means you need to `yarn add @nomiclabs/hardhat-ethers` otherwise `@openzeppelin/hardhat-upgrades` won't work properly (most likely)

**Review by Remscar — APPROVED** [review]
Once those 2 minor changes are done, this can be merged. Great work!

**Comment by SalvoAgustin** [comment]
Done changes, deleted unnecessary space, and added required dependencies

**Pull request merged**